doc.txt

```diff
@@ -1,11 +1,30 @@
 /**
+ * @defgroup utils Utilities
+ * @brief Utilities and helper functionality
+ *
+ *        Additional scripts and configuration options to ease RIOT
+ *        development.  RIOT ecosystem.
```

Comment:
The phrase "RIOT ecosystem" is not a sentence, did you accidentally delete something you wanted to put there?